sources/subs/Attachments.subs.php

```diff
@@ -319,10 +319,11 @@ function createAttachment(&$attachmentOptions)
 	if ($is_image)
 	{
 		$attachmentOptions['mime_type'] = getValidMimeImageType($size[2]);
+
 		// Want to correct for phonetographer photos?
 		if (!empty($modSettings['attachment_autorotate']))
 		{
-			$image->autoRotate();
+			$image->autoRotateImage();
```

Comment:
lol
I guess I used autoRotate on the assumption that "Image" was implicit since the object is an image, then I forgot to update the method name in the class. :man_facepalming: 
Oh well, autoRotateImage is fine either.
Thanks for the catch!!

Reply:
Easy to rename, not like we are done with attachments just yet ;)